src/main/resources/jobConfig.yml

```diff
@@ -1742,7 +1742,7 @@ Jobs:
         Name: "Regular Joe"
         Objectives:
         - Breed;Sheep;10
-        - Shear;White;10
+        - Shear;WHITE;10

```

Comment:
Revert this. This needs some improvements in the code that the material names should be `lowerCase` even the user uses large letter.

Reply:
done!